sdk/storage/Azure.Storage.Blobs/src/PageBlobClient.cs

```diff
@@ -188,7 +188,14 @@ public PageBlobClient(Uri blobUri, TokenCredential credential, BlobClientOptions
             ClientDiagnostics clientDiagnostics,
             CustomerProvidedKey? customerProvidedKey,
             string encryptionScope)
-            : base(blobUri, pipeline, version, clientDiagnostics, customerProvidedKey, encryptionScope)
+            : base(
```

Comment:
Why do we throw in the toBlockBlobClient method but not in PageBlob/AppendBlob? Did I miss something?

Reply:
Added to the extension methods at the bottom of the files, as in blockblobclient. Probably have to address this with public constructors as well.

Reply:
Should the TypeNotSupported method pass in typeof(PageBlobClient)? Otherwise looks good.